openapi/openapi.yml

```
@@ -3,31 +3,604 @@ info:
+        '200':
+          $ref: '#/components/responses/UpdateSuccessRequestResponse' 
+        '404':
+          $ref: '#/components/responses/NotFoundResponse'
+        '500':
+          $ref: '#/components/responses/InternalServerErrorResponse'
+  /users/others-change/{user_id}:
+    post:
+      summary: 他情報変更
+      description: 年齢、都道府県、性別の変更を行う。
+      tags:
+        - users
+      security:
+      - Bearer: []
+      parameters:
+      - $ref: '#/components/parameters/userid'
+      requestBody:
+        $ref: '#/components/requestBodies/OthersChangeRequestBody'
+      responses:
+        '200':
+          $ref: '#/components/responses/UpdateSuccessRequestResponse' 
+        '401':
+          $ref: '#/components/responses/UnauthorizedResponse'
+        '404':
+          $ref: '#/components/responses/NotFoundResponse'
+        '500':
+          $ref: '#/components/responses/InternalServerErrorResponse'
+
+  /arts:
+    get:
+      summary: 作品を取得する。
+      description: すべての作品を取得する。
+      tags:
+        - arts
+      responses:
+        '200':
+          $ref: '#/components/responses/ArtsResponse'
+        '404':
+          $ref: '#/components/responses/NotFoundResponse'
+        '500':
+          $ref: '#/components/responses/InternalServerErrorResponse'
+  /arts/author:
+    get:
+      summary: 作者の作品を取得する。
+      description: 作者の「あいうえお」順にして返す。
+      tags:
+        - arts
+      responses:
+        '200':
+          $ref: '#/components/responses/ArtsResponse'
+        '404':
+          $ref: '#/components/responses/NotFoundResponse'
+        '500':
+          $ref: '#/components/responses/InternalServerErrorResponse'
+  /arts/save/{user_id}:
+    get:
+      summary: ユーザが保存した作品を取得する。
+      description: ユーザIDを渡し、保存した作品を返す。
+      tags:
+        - arts
+      security:
+      - Bearer: []
+      parameters:
+      - $ref: '#/components/parameters/userid'
+      responses:
+        '200':
+          $ref: '#/components/responses/ArtsResponse'
+        '401':
+          $ref: '#/components/responses/UnauthorizedResponse'
+        '404':
+          $ref: '#/components/responses/NotFoundResponse'
+        '500':
+          $ref: '#/components/responses/InternalServerErrorResponse'
+
+  /stamps/{user_id}:
+    get:
+      summary: スタンプを取得する。
+      description: ユーザのスタンプを地域ごとに取得する。
+      tags:
+        - stamps
+      security:
+      - Bearer: []
+      parameters:
+      - $ref: '#/components/parameters/userid'
+      responses:
+        '200':
+          $ref: '#/components/responses/StampsResponse'
+        '401':
+          $ref: '#/components/responses/UnauthorizedResponse'
+        '404':
+          $ref: '#/components/responses/NotFoundResponse'
+        '500':
+          $ref: '#/components/responses/InternalServerErrorResponse'
+
+components:
+  parameters:
+    userid:
+      name: userId
+      in: path
+      description: ユーザID
+      required: true
+      schema:
+        type: string
+        example: 12345     
```

これだとnumberなので

```suggestion
        example: "12345"  
```

```suggestion
        example: "hello world"  
```

くらいにしておきましょうか